### DISCUSSION TITLE
Join when one task from the total of 2 forked tasks finishes

### DISCUSSION THREAD

**Discussion by ardeleana** [Netflix/conductor#3718] [General]
Hi there!

Is there a way to  make join completed when one task is finished from the totall of 2 forked tasks?
I am interested in something like an OR condition.
fork:
 [task1,
task2]
join complete when either task1, either task2 completes

**Comment by manan164** [comment]
Hi @ardeleana , Currently it is not supported. Feel free to raise PR with changes.  However, we at Orkes do support it in the enterprise version of Conductor. We can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) for more realtime collaboration
